lego/templates/email/base.html

```diff
@@ -1,310 +1,273 @@
-
-<table class="body-wrap">
-	<tr>
-		<td></td>
-		<td class="container" width="600">
-			<div class="content">
-				<table class="main" width="100%" cellpadding="0" cellspacing="0" itemprop="action" itemscope itemtype="http://schema.org/ConfirmAction">
-
-                    {% block alert %}{% endblock %}
-
-                    <tr>
-						<td class="content-wrap">
-							<meta itemprop="name" content="{{ title }}"/>
-							<table width="100%" cellpadding="0" cellspacing="0">
-
-                                {% block content %}{% endblock %}
-
-								<tr>
-									<td class="content-block">
-										&mdash; {{ from }}
-									</td>
-								</tr>
-							</table>
-						</td>
-					</tr>
-				</table>
-				<div class="footer">
-					<table width="100%">
-						<tr>
-							<td class="aligncenter content-block">
-                               Denne e-posten ble sendt fra {{ system_name }}, systemet som driver <a href="{{ frontend_url }}">{{ frontend_url }}</a>.
-                            </td>
-						</tr>
-					</table>
-				</div></div>
-		</td>
-		<td></td>
-	</tr>
+<body bgcolor="#ffffff">
+<table align="center" cellpadding="0" cellspacing="0" class="container-for-gmail-android" width="100%">
+  <tr>
+    <td align="left" valign="top" width="100%" style="background:repeat-x url(http://s3.amazonaws.com/swu-filepicker/4E687TRe69Ld95IDWyEg_bg_top_02.jpg) #ffffff;">
+      <center>
+      <img src="http://s3.amazonaws.com/swu-filepicker/SBb2fQPrQ5ezxmqUTgCr_transparent.png" class="force-width-gmail">
+        <table cellspacing="0" cellpadding="0" width="100%" bgcolor="#ffffff" background="http://s3.amazonaws.com/swu-filepicker/4E687TRe69Ld95IDWyEg_bg_top_02.jpg" style="background-color:transparent">
+          <tr>
+            <td width="100%" height="80" valign="top" style="text-align: center; vertical-align:middle;">
+            <!--[if gte mso 9]>
+            <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="mso-width-percent:1000;height:80px; v-text-anchor:middle;">
+              <v:fill type="tile" src="http://s3.amazonaws.com/swu-filepicker/4E687TRe69Ld95IDWyEg_bg_top_02.jpg" color="#ffffff" />
+              <v:textbox inset="0,0,0,0">
+            <![endif]-->
+              <center>
+                <table cellpadding="0" cellspacing="0" width="600" class="w320">
+                  <tr>
+                    <td class="pull-left mobile-header-padding-left" style="vertical-align: middle;">
+                      <a href=""><img width="150" height="30" src="http://127.0.0.1:8000/static/assets/img/logo.png" alt="logo"></a>
+                    </td>
+                    <td class="pull-right mobile-header-padding-right" style="color: #4d4d4d;">
+                      <a href="https://www.facebook.com/groups/398146436914007/"><img width="38" height="47" src="http://s3.amazonaws.com/swu-filepicker/LMPMj7JSRoCWypAvzaN3_social_09.gif" alt="facebook" /></a>
```

Comment:
This should link to the official facebook page for Abakus, not the internal group. Also, it would be nice to have links to Instagram (and LinkedIn?) as well :)